Projects/b_u585i_iot02a_tfm/Src/app_main.c

```diff
@@ -122,24 +122,22 @@ void vInitTask( void * pvArgs )
     xResult = xTaskCreate( &net_main, "MxNet", 1024, NULL, 23, NULL );
     configASSERT( xResult == pdTRUE );
 
-    xResult = xTaskCreate( vMQTTAgentTask, "MQTTAgent", 2048, NULL, tskIDLE_PRIORITY + 1, NULL );
+    xResult = xTaskCreate( vMQTTAgentTask, "MQTTAgent", 2048, NULL, tskIDLE_PRIORITY + 3, NULL );
     configASSERT( xResult == pdTRUE );
 
-/*
- *  xResult = xTaskCreate( vOTAUpdateTask, "OTAUpdate", 4096, NULL, tskIDLE_PRIORITY + 1, NULL );
- *  configASSERT( xResult == pdTRUE );
- */
+    xResult = xTaskCreate( vOTAUpdateTask, "OTAUpdate", 2048, NULL, tskIDLE_PRIORITY + 3, NULL );
+    configASSERT( xResult == pdTRUE );
 
-    xResult = xTaskCreate( vEnvironmentSensorPublishTask, "EnvSense", 1024, NULL, 10, NULL );
+    xResult = xTaskCreate( vEnvironmentSensorPublishTask, "EnvSense", 1024, NULL, tskIDLE_PRIORITY + 2, NULL );
```

Comment:
Is tskIDLE_PRIORITY really necessary?

Reply:
Will do this in a follow up PR after fixing the task priorities of all tasks.